cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -31,10 +33,71 @@
 import static com.powsybl.cgmes.conversion.Conversion.Config.StateProfile.SSH;
 
 /**
+ * TwoWindingsTransformer Interpretation
+ * <p>
+ * Ratio and Phase Interpretation (Xfmr2RatioPhaseInterpretationAlternative) <br>
+ * END1. All tapChangers (ratioTapChanger and phaseTapChanger) are considered at end1 (before transmission impedance) <br>
```

Comment:
Could you be more accurate: by ````end1```` do you mean ````PowerTransformerEnd```` ?

Reply:
A two windings transformer is connected to 2 voltage levels (side 1 and side 2) that belong to the same substation. end1 means side 1 (usually high voltage) and end2 means side 2 (usually the low voltage). 

At the interpretation stage we use a two windings transformer model more general than the IIDM model. This interpreted model supports `tapChangers` at both sides.

If the alternative option is END1  `ratioTapChanger` and `phaseTapChanger` are always mapped to side 1 of the `interpretedTwoWindingsTransformerModel`  (high voltage level usually) . It does not matter in which `CgmesTransformerEnd` the `tapChanger` is defined.

If the alternative option is END2  `ratioTapChanger` and `phaseTapChanger` are always mapped to side 2 of the `interpretedTwoWindingsTransformerModel` (low voltage level usually) . It does not matter in which `CgmesTransformerEnd` the `tapChanger` is defined.

if the alternative option is END1_END2  the side to be mapped depends on the `CgmesTransformerEnd` where the  `tapChanger` is defined. `tapChanger`  is mapped to side 1 if it is defined at `CgmesTransformerEnd1`  and it is mapped to side 2 if it is defined at `CgmesTransformerEnd2`.

Finally, in the last option (X) the side is selected depending on the `CgmesTransformerEnd` where the transformer reactance (X) is defined.  
If reactance is defined at `CgmesTransformerEnd2` (X1 == 0.0 and X2 != 0.0) then the `tapChanger` is mapped to side 1.
If reactance is defined at `CgmesTransformerEnd1` (X1 != 0.0 and X2 == 0.0) then the `tapChanger` is mapped to side 2.
This convention is used by one TSO.

After the interpretation stage the `interpretedTwoWindingsTransformerModel` is converted to the `iidmTwoWindingsTransformerModel`